commit 65ff45b33cabbdd31fb461131437cc741336074b
Author: Christoph Hellwig <hch@lst.de>
Date:   Mon Feb 20 17:04:42 2017 -0500

    nfsd: minor nfsd_setattr cleanup
    
    commit 758e99fefe1d9230111296956335cd35995c0eaf upstream.
    
    Simplify exit paths, size_change use.
    
    Signed-off-by: Christoph Hellwig <hch@lst.de>
    Signed-off-by: J. Bruce Fields <bfields@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1331_65ff45b33cabbdd31fb461131437cc741336074b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1331_65ff45b33cabbdd31fb461131437cc741336074b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
